src/parsers/manifest/local/types.ts

```diff
@@ -166,7 +166,7 @@ export interface ILocalIndex {
 
 /** A quality for a given "local" Manifest track (`ILocalAdaptation`). */
 export interface ILocalRepresentation {
-  /** Average number of bytes per second for the segments contained in this quality. */
+  /** Maximum number of bytes per second for the segments contained in this quality. */
```

Comment:
Are we sure ?

Reply:
I checked, and that's actually the case with DASH contents at least.

I think it makes more sense to be as compatible as possible to our ABR logic.